pkg/utils/crtl_utils.go

```diff
@@ -179,3 +207,41 @@ func CalculateDuration(creationTime time.Time, finishTime time.Time) string {
 	}
 	return finishTime.Sub(creationTime).Round(time.Second).String()
 }
+
+func GenerateRandomRequeueDurationFromEnv() (needReconcile bool, d time.Duration) {
+	d = defaultRuntimeReconcileDuration
+	needReconcile = true
+
+	if RuntimeReconcileDurationEnvVal == "" {
+		return
+	}
+
+	duration, err := strconv.Atoi(RuntimeReconcileDurationEnvVal)
+	if err != nil {
+		return
+	}
+
+	if duration == -1 {
+		needReconcile = false
+		return
+	}
+
+	d = time.Duration(duration) * time.Second
+
+	if RuntimeReconcileDurationOffsetEnvVal == "" {
+		return
+	}
+
+	offset, err := strconv.Atoi(RuntimeReconcileDurationOffsetEnvVal)
+	if err != nil {
+		return
+	}
+
+	if offset < 0 || offset > duration {
+		return
+	}
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
+	randomDurationValue := (r.Intn(2*offset+1) + duration - offset)
```

Comment:
Why not reusing the global math/rand instance to avoid repeatedly creating rand.Source？Reuse the global math/rand instance directly to avoid repeatedly creating rand.Source and rand.Rand.